amacc.c

```diff
@@ -408,7 +408,10 @@ void expr(int lev)
             sz = t >= PTR2 ? sizeof(int) :
                              t >= PTR ? tsize[t - PTR] : 1;
             if (t == ty && sz > 1) {
-                *++e = SUB; *++e = PSH; *++e = IMM; *++e = sz; *++e = SUB;
+                switch(sz) {
```

Comment:
There should be a space before the open parenthesis. That is, `switch (sz) {` rather than `switch(sz) {`.